recipes/srnaminer/meta.yaml

```diff
@@ -37,7 +37,7 @@ test:
 
 about:
   home: https://github.com/kli28/sRNAminer
-  license: GPL3
+  license: MIT
```

Comment:
This conflicts with your readme, which states:

> sRNAminer is a platform-independent software that can be run under all operating systems. lt is freely available > to non-commercial users at https://github.com/kli28/sRNAminer/releases

Please clarify the license terms, as the MIT license grants even commercial users the right to use your code, which conflicts with what you say in the readme.